lesson3.md

```diff
@@ -0,0 +1,837 @@
+
+Which is more understandable?
+
+```js
+42 > 22 && 42 > 35
+```
+
+Or
+
+```js
+let myAge = 42;
+let leftPersonAge = 22;
+let rightPersonAge = 35;
+
+let olderThanLeft = myAge > leftPersonAge;
+let olderThanRight = myAge > rightPersonAge;
+let isOldest = olderThanLeft && olderThanRight;
+```
+
+
+---
+
+
+### Quick Quiz, now in VSCode!
+
+Let's say, everyone here wants to go to the cinema.
+
+How many people in total are going to the cinema?
+<!-- .element: class="fragment" -->
+
+A ticket to watch the movie *The Joker* costs 8€, how much do we have to pay in total?
+<!-- .element: class="fragment" -->
+
+Saskia and I volunteered to pay for the tickets, how much each one pays?
+<!-- .element: class="fragment" -->
+
+
+---
+
+
+### Quick Quiz (Cont.)
+
+
+I can only pay 100€, is it enough?
+
+
+The cinema will only allow us to enter if we are less than 30 people and if we pay more than 100€, will we get in?
+<!-- .element: class="fragment" -->
+
+
+---
+
+
+### Recap
+
+| Group | Operators | Example |
+| ----- | --------- | ------- |
+| Numerical Operators | `+ - * /` | `5 + 4 * 3`<br/>`7 / 2 - 2`<br/>`"Hello" + " World"` |
+| Comparison Operators | `=== !== < > <= >=` | `temperature !== 25`<br/>`age >= 18` |
+| Logical Operators | &#124;&#124; `&& !` | `a && !b`<br />`x >= 5 && x < 15` |
+| **Combined Operators** | `+= -= *= /= ++ --` | `a *= 2`<br />`count++` |
+
+
+---
+
+
+### Excursus: Combined Operators
+
+This
+
+```js
+let myAge = 42;
+myAge = myAge + 5;
+```
+
+Is the same as this
+
+```js
+let myAge = 42;
+myAge += 5;
+```
+
+
+---
+
+
+### Quiz: What's the price?
+
+```js
+let price = 10;
+
+// there's a 50 percent sale!
+price /= 2;
+
+// price is ??
+```
+
+Result: `5`
+<!-- .element: class="fragment" -->
+
+
+---
+
+
+### Operator ++ and --
+
+Operator `++` and `--` increase (or decrease) the value of a variable by 1
+
+```js
+let myAge = 42;
+myAge++; // same as myAge = myAge + 1
+```
+
+
+---
+
+
+<!-- .slide: id="lesson3:if" -->
+# Conditional Statements
+
+
+---
+
+
+### Conditional Statements
+
+Up until now, we wrote code line by line.
+
+But what if I want to run some code only if the user is logged in?
+
+
+---
+
+
+### If statement
+
+```js
+if (condition) {
+  // block of code that
+  // will run ONLY if
+  // condition is true
+}
+```
+
+
+---
+
+
+### If statements
+
+```js
+if (condition) {
+  // your code here
+}
+```
+
+* `condition` must be a boolean
+* The code in the block is __only__ executed if the boolean is `true`
+* If the boolean is false, the code block is _not_ executed
+
+
+---
+
+
+### if Statements - Examples
+
+```js
+if (food === 'pizza') {
+  console.log('My favorite!');
+}
+```
+
+```js
+if (money < 1000000) {
+  console.log('You are not rich :p');
+}
+```
+
+```js
+if (canSpeakRussian) { // same as: canSpeakRussian === true
+  console.log('Привет!');
+}
+```
+
+
+---
+
+
+### Code Blocks: {}
+
+Code blocks is the code between `{` and `}`
+
+Should be indented for better readability:
```

Comment:
Since these are kinda incomplete sentences, do you want to make them bullet points instead?